tests/ui/non_expressive_names.rs

```diff
@@ -142,6 +142,11 @@ fn underscores_and_numbers() {
     let _1_ok= 1;
 }
 
+fn issue2927() {
```

Comment:
This example does not acctually replicate the issue: http://play.rust-lang.org/?gist=48e4192bd06e619f1c13858e591ffd45&version=stable&mode=debug&edition=2015

Reply:
The lint is allow by default. Here is a fixed version: http://play.rust-lang.org/?gist=b247b042cb0407db292ba9d501bd6d1a&version=stable&mode=debug&edition=2015